app/Http/Controllers/VueController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class VueController extends Controller
```

Comment:
Why do we need this controller? The rendering of register form should be done through RegisterController.